app/src/main/java/nl/eduvpn/app/inject/ApplicationModule.java

```diff
@@ -104,10 +104,23 @@ protected SerializerService provideSerializerService() {
 
     @Provides
     @Singleton
-    protected VPNService provideVPNService(Context context, PreferencesService preferencesService) {
-        return new VPNService(context, preferencesService);
+    protected EduOpenVPNService provideEduOpenVPNService(Context context, PreferencesService preferencesService) {
+        return new EduOpenVPNService(context, preferencesService);
     }
 
+    /*
```

Comment:
If you won't use this anymore please remove it.
If you will, please add a comment why it is commented, and when/how it will be reinstated.

Reply:
Fixed in https://github.com/eduvpn/android/pull/318/commits/683e60176399a7a8fe8c805e6a35db97f98418e3